solver/src/solver/http_solver.rs

```diff
@@ -334,12 +368,15 @@ impl HttpSolver {
         let limit_orders = self.map_orders_for_solver(orders);
         let constant_product_orders = self.map_amm_orders_for_solver(constant_product_liquidity);
         let weighted_product_orders = self.map_amm_orders_for_solver(weighted_product_liquidity);
+        let stable_pool_orders = self.map_amm_orders_for_solver(stable_pool_liquidity);
+
```

Comment:
This was tested locally by putting a print statement here:

```
println!("Stable Pool Orders {:?}", stable_pool_orders.len());
```
and placing an order of `USDC <-> DAI` which is a known [Balancer stable pool.](https://etherscan.io/address/0x06df3b2bbb68adc8b0e302443692037ed9f91b42)